src/digitalIdentity-frontend/src/app/create-schema-page/create-schema-page.component.ts

```diff
@@ -222,12 +227,84 @@ export class CreateSchemaPageComponent implements OnInit {
         this.schemaTmp.attributes[i].attribID;
       this.schema.attributes[i].value = this.schemaTmp.attributes[i].value;
     }
-    for (
-      let i = 0;
-      i < this.schema.attributes.length - this.schemaTmp.attributes.length;
-      i++
-    ) {
+
+    let maxi: any =
+      this.schema.attributes.length - this.schemaTmp.attributes.length;
+    for (let i = 0; i < maxi; i++) {
       this.schema.attributes.pop();
     }
+    this.postSchema();
+  }
+
+  postSchema(): void {
+    const headers = new HttpHeaders().append(
+      'Content-Type',
+      'application/json'
+    );
+    let body = JSON.stringify(this.schema);
+    let params = this.schemaToHttpParams(this.schema);
+
+    this.http
+      .post<any>(environment.serverURL + '/schemas/create', body, {
+        headers: headers,
+        observe: 'response',
+        params: params,
+      })
+      .subscribe({
+        next: (response) => {
+          if (response.status == 201) {
+            this.router.navigate(['/schema-overview']);
+          } else {
+            this.openDialog(
+              'Creation not successful!',
+              'Server response: ' + response.body
+            );
+
+            if (isDevMode()) {
+              console.log(
+                'Creation not successful! Server response: ' + response.body
+              );
+            }
+          }
+        },
+        error: (error) => {
+          this.openDialog(
+            'Creation not successful! Server response!',
+            'Server response: ' + error.status + ' ' + error.message
+          );
+          console.log(error);
+        },
+      });
+  }
+
+  schemaToHttpParams(schema: schema): HttpParams {
+    let params: HttpParams = new HttpParams();
+    params = params.append('authorization', 'passing');
+    params = params.append('alias', schema.name);
+    params = params.append('version', schema.version);
+    // params = params.append('attributes', JSON.stringify(schema.attributes));
```

Comment:
Is this commented line useful?

Reply:
it might be reimplemented if the backend is able to accept ["test","attribut","attr3"]